sled-agent/src/params.rs

```diff
@@ -825,3 +817,20 @@ pub struct CleanupContextUpdate {
     /// The new limit on the underlying dataset quota allowed for bundles.
     pub storage_limit: Option<u8>,
 }
+
+// Our SledRole and Baseboard types do not have to be identical to the Nexus
+// ones, but they generally should be, and this avoids duplication.  If it
+// becomes easier to maintain a separate copy, we should do that.
+pub type SledRole = nexus_client::types::SledRole;
```

Comment:
Is this going to make the build even longer?

It feels somewhat wrong to me to depend on a nexus client type here, but I'm not completely horrified. I'm sure we do this elsewhere.

Reply:
Works for me.